Error While applying Notes RDBMS Liquibase Changelog

### Problem Statement

While starting server with existing data, a changelog error is logged:
```
2025-12-11 04:32:14,337 | ERROR | ChangeSet db/changelog/wiki.db.changelog-1.0.0.xml::1.0.0-54::wiki encountered an exception. [liquibase.changelog.ChangeSet<Catalina-utility-14>] 
liquibase.exception.DatabaseException: Duplicate entry 'nessrineberrama-user' for key 'WIKI_WIKIS.UK_WIKI_WIKIS_OWNER_TYPE_01' [Failed SQL: (1062) ALTER TABLE community.WIKI_WIKIS ADD CONSTRAINT UK_WIKI_WIKIS_OWNER_TYPE_01 UNIQUE (OWNER, TYPE)]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:497)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:83)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:185)
	at liquibase.executor.AbstractExecutor.execute(AbstractExecutor.java:141)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1189)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:790)
	at liquibase.changelog.visitor.UpdateVisitor.executeAcceptedChange(UpdateVisitor.java:126)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:70)
	at liquibase.changelog.ChangeLogIterator.lambda$run$0(ChangeLogIterator.java:133)
	at liquibase.Scope.lambda$child$0(Scope.java:201)
	at liquibase.Scope.child(Scope.java:210)
	at liquibase.Scope.child(Scope.java:200)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.changelog.ChangeLogIterator.lambda$run$1(ChangeLogIterator.java:122)
	at liquibase.Scope.lambda$child$0(Scope.java:201)
	at liquibase.Scope.child(Scope.java:210)
	at liquibase.Scope.child(Scope.java:200)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:267)
	at liquibase.Scope.child(Scope.java:271)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:91)
	at liquibase.command.core.AbstractUpdateCommandStep.lambda$run$0(AbstractUpdateCommandStep.java:114)
	at liquibase.Scope.lambda$child$0(Scope.java:201)
	at liquibase.Scope.child(Scope.java:210)
	at liquibase.Scope.child(Scope.java:200)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.command.core.AbstractUpdateCommandStep.run(AbstractUpdateCommandStep.java:112)
	at liquibase.command.core.UpdateCommandStep.run(UpdateCommandStep.java:100)
	at liquibase.command.CommandScope.lambda$execute$6(CommandScope.java:253)
	at liquibase.Scope.child(Scope.java:210)
	at liquibase.Scope.child(Scope.java:186)
	at liquibase.command.CommandScope.execute(CommandScope.java:241)
	at liquibase.Liquibase.lambda$update$0(Liquibase.java:216)
	at liquibase.Scope.lambda$child$0(Scope.java:201)
	at liquibase.Scope.child(Scope.java:210)
	at liquibase.Scope.child(Scope.java:200)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Liquibase.runInScope(Liquibase.java:1333)
	at liquibase.Liquibase.update(Liquibase.java:205)
	at liquibase.Liquibase.update(Liquibase.java:188)
	at liquibase.Liquibase.update(Liquibase.java:175)
	at liquibase.Liquibase.update(Liquibase.java:165)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.applyChangeLog(LiquibaseDataInitializer.java:211)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.initData(LiquibaseDataInitializer.java:193)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.initData(LiquibaseDataInitializer.java:155)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.start(LiquibaseDataInitializer.java:137)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.exoplatform.container.LifecycleVisitor.traverse(LifecycleVisitor.java:115)
	at org.exoplatform.container.LifecycleVisitor.start(LifecycleVisitor.java:181)
	at org.exoplatform.container.ConcurrentContainer.start(ConcurrentContainer.java:475)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:64)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:64)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:64)
	at org.exoplatform.container.ExoContainer.start(ExoContainer.java:286)
	at org.exoplatform.container.ExoContainer.start(ExoContainer.java:267)
	at org.exoplatform.container.RootContainer.buildRootContainer(RootContainer.java:831)
	at org.exoplatform.container.RootContainer.getInstance(RootContainer.java:907)
	at org.exoplatform.web.GenericHttpListener.contextInitialized(GenericHttpListener.java:155)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4046)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4474)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:655)
	at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:965)
	at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1903)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'nessrineberrama-user' for key 'WIKI_WIKIS.UK_WIKI_WIKIS_OWNER_TYPE_01'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:109)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:113)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:829)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:681)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:123)
	at jdk.proxy3/jdk.proxy3.$Proxy22.execute(Unknown Source)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:491)
	... 73 common frames omitted
```

### Additional Context

_No response_

--- a/notes-service/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml
+++ b/notes-service/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml
@@ -455,6 +455,50 @@
       columnDataType="TIMESTAMP"
       defaultNullValue="${now}" />
   </changeSet>
+  
+  <changeSet id="1.0.0-53-clean-pages" author="wiki">
+    <sql>
+		DELETE FROM WIKI_PAGES
+		WHERE WIKI_ID IN (
+		  SELECT w1.WIKI_ID
+		  FROM WIKI_WIKIS w1
+		  JOIN WIKI_WIKIS w2
+			ON w1.OWNER = w2.OWNER
+		   AND w1.TYPE = w2.TYPE
+		   AND w1.WIKI_ID > w2.WIKI_ID
+		)
+    </sql>
+  </changeSet>
+  
+  <changeSet id="1.0.0-53-clean-templates" author="wiki">
+    <sql>
+		DELETE FROM WIKI_TEMPLATES
+		WHERE WIKI_ID IN (
+		  SELECT w1.WIKI_ID
+		  FROM WIKI_WIKIS w1
+		  JOIN WIKI_WIKIS w2
+			ON w1.OWNER = w2.OWNER
+		   AND w1.TYPE = w2.TYPE
+		   AND w1.WIKI_ID > w2.WIKI_ID
+		)
+    </sql>
+  </changeSet>
+  
+  <changeSet id="1.0.0-53-clean-wikis" author="wiki">
+    <sql>
+		DELETE FROM WIKI_WIKIS
+        WHERE WIKI_ID IN (
+          SELECT WIKI_ID FROM (
+            SELECT w1.WIKI_ID
+            FROM WIKI_WIKIS w1
+            JOIN WIKI_WIKIS w2
+            ON w1.OWNER = w2.OWNER
+            AND w1.TYPE = w2.TYPE
+            AND w1.WIKI_ID > w2.WIKI_ID
+          ) dup
+        );
+    </sql>
+  </changeSet>
 
   <changeSet id="1.0.0-54" author="wiki" failOnError="false">
     <addUniqueConstraint columnNames="OWNER, TYPE" tableName="WIKI_WIKIS"

